docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: "3"
+services:
+  vast:
+    container_name: tenzir-vast
+    image: tenzir/vast:${VAST_VERSION:-latest}
```

Comment:
I observed that even if we provide "image:..", docker-compose always starts building rather than pulling. This is not an expected behaviour, I wouldnt expect a docker-compose file in root folder to directly build when latest release is in DockerHub. We can put it back but it will increase the CI and local debugging time.

Let me know if there are alternatives we can use in docker-compose to fix this behaviour.

Reply:
If you put the `pull_policy: build` alongside the `build` into an override file for building you get the out-of-the-box experience that we were looking for: By default you pull, and you can opt-in to building by using the build file.

Reply:
pull_policy also affects `docker run` and everytime tests try to `docker compose run` a VAST client, it tries to pull/build.

I removed the build instructions from docker-compose and included them in documentation if anyone needs to build. Behaviour seems to have side effects and complicates.

If you think we should lean towards this a bit more, I can check further. `docker-compose` should be easy. Building is an optional action most likely to be taken by developers/contributors.

Reply:
> If you put the pull_policy: build alongside the build into an override file for building you get the out-of-the-box experience that we were looking for: By default you pull, and you can opt-in to building by using the build file.

Please read again—I actually got it to work as expected with this.